test/static/expected_metadata_metadataExtensions.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<EntityDescriptor xmlns="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" entityID="http://example.serviceprovider.com" ID="d700077e-60ad-49c1-b93a-dd1753528708">
```

Comment:
If you look at page 6 of the spec, it seems you might be missing a namespace declaration for `md:*` properties: e.g. `xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata`.

Reply:
You're are right.
I fixed it